pkg/infrastructure/services/image.go

```diff
@@ -175,7 +177,7 @@ func NewImageHandlerForBuiltServices(image *v1alpha1.Image, namespace string, cl
 	}
 	if cli.IsOpenshift() {
 		// creates the empty tag reference in the ImageStream since this handler is for services being built
-		handler.createImageStream(namespace, false)
+		handler.createImageStream(namespace, false, false)
```

Comment:
@sutaakar I believe we would need this information here as well since this is the target `ImageStream` for the `KogitoRuntime` when the image is being built in the cluster. You might need to add this flag in the `KogitoBuild` CR as well to have it here.

Reply:
Ignore if the flag is only needed when pulling from an external registry, in the use cases where the image is built and pushing to a third party registry instead.

Reply:
Yes, the flag is used just for pulling.
Here we directly push image to image stream so there is no need for the flag.